src/components/Splash.js

```diff
@@ -144,8 +159,33 @@ export default ({
   backgroundColor = theme.primary,
   page = `TORONTO`,
 }: Props) => {
+
+  useEffect(() => {
+    const guildCardRoot = document.getElementById(`guild-card-root`)
+    const guildCardUpcoming = document.getElementById(`guild-card-upcoming`)
+    const guildCardPast = document.getElementById(`guild-card-past`)
+
+    window.$guild.renderGuildCard(
+      guildCardRoot,
+      `torontojs`
+    )
+
+    window.$guild.renderGuildEventList(
+      guildCardUpcoming,
+      `torontojs`,
+      `UPCOMING`
+    )
+
+    window.$guild.renderGuildEventList(
+      guildCardPast,
+      `torontojs`,
+      `PAST`
+    )
+  })
```

Comment:
We only need to do this on mount
```suggestion
  }, [])
```

Reply:
I missed that. Make sense 👍